src/components/Input/Input.tsx

```diff
@@ -16,13 +16,28 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import {
+  ChangeEvent,
+  ReactElement,
+  useCallback,
+  useMemo,
+  useState,
+} from 'react'
 import { Input as AntInput } from 'antd'
-import { ReactElement } from 'react'
 
+import { AddonType, Type } from './types'
 import { BaseInput, defaults as baseInputDefaults } from '../BaseInput/BaseInput'
+import { InputAddon, InputAddonProps } from './InputAddon.tsx'
 import { Icon } from '../Icon'
 import { InputProps } from './props'
-import { Type } from './types'
+import styles from './input.module.css'
+
+/**
+ * A UI element to insert text content in a form.
+ *
+ * @link https://ant.design/components/input
+ * @returns {Input} Input component
+ */
```

Comment:
why did you move this JSDoc? It should be just above the `Input` declaration, right?